pkg/logs/input/util.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package input
+
+import (
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/tagger"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// taggerFunc purpose is to ease testing ServiceFromTags
+var taggerFunc = tagger.StandardTags
+
+// ServiceFromTags returns the standard tag 'service' corresponding to a container
+// It returns an empty string if tag not found
+func ServiceFromTags(ctrName, taggerEntity string) string {
```

Comment:
same nit here, I'd rename to:

```suggestion
func ServiceNameFromTags(ctrName, taggerEntity string) string {
```